soroban-env-host/Cargo.toml

```diff
@@ -40,9 +43,10 @@ testutils = []
 
 [target.'cfg(target_os = "linux")'.dev-dependencies]
 perf-event = "0.4.7"
-tracking-allocator = "0.4.0"
-tabwriter = "1.2.1"
-thousands = "0.2.0"
+
+[target.'cfg(all(target_os = "macos", target_arch = "aarch64"))'.dev-dependencies]
+macos-perf = "0.1.1"
+sudo = "0.6.0"
```

Comment:
The description of the `sudo` crate is terrifying. From the docs.rs:
>Detect if you are running as root, restart self with sudo if needed or setup uid zero when running with the SUID flag set.

As far as I can tell we're only using the detection mechanism, and not the auto-restart as sudo. Right?

Reply:
Yeah. I think I will actually be able to roll this change back and use getrusage_v4 instead, but as it stands this is "use for detection only" and "only in a benchmark" -- no need to be alarmed.